plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -223,7 +223,7 @@ func clusterRoles() []rbacv1.ClusterRole {
 			ObjectMeta: metav1.ObjectMeta{Name: "system:openshift:public-info-viewer"},
 			Rules: []rbacv1.PolicyRule{
 				rbacv1helpers.NewRule("get").URLs(
-					"/.well-known", "/.well-known/*",
+					"/.well-known", "/.well-known/oauth-authorization-server",
```

Comment:
we carry a patch on kube policy today?  If not (and I thought I avoided this), create a new clusterrole and clusterrolebinding in our kube-apiserver operator.

Reply:
indeed, this would be the carry https://github.com/openshift/kubernetes/commit/f271c4722113469570f7fe9834ac0884bf46fe8a